app/frontend/src/application/jobseekers/map.js

```diff
@@ -87,7 +100,44 @@ window.initMap = () => {
       infoWindow.close();
     });
 
-    /* eslint-enable */
+  } else if (polygonBoundaries == "no polygons") {
```

Comment:
If no polygons are used in the search can we just not show a map? Every little helps 💰

Reply:
Good point, I was over-eager